packages/server/src/api/controllers/user.js

```diff
@@ -20,27 +20,21 @@ exports.fetch = async function(ctx) {
 
 exports.create = async function(ctx) {
   const db = new CouchDB(ctx.user.appId)
-  const {
-    username,
-    password,
-    name,
-    accessLevelId,
-    permissions,
-  } = ctx.request.body
+  const { email, password, name, accessLevelId, permissions } = ctx.request.body
 
-  if (!username || !password) {
-    ctx.throw(400, "Username and Password Required.")
+  if (!email || !password) {
+    ctx.throw(400, "email and Password Required.")
   }
 
   const accessLevel = await checkAccessLevel(db, accessLevelId)
 
   if (!accessLevel) ctx.throw(400, "Invalid Access Level")
 
   const user = {
-    _id: generateUserID(username),
-    username,
+    _id: generateUserID(email),
+    email,
     password: await bcrypt.hash(password),
-    name: name || username,
```

Comment:
Related to my previous comment, now that name is simply passed in there will be cases where a user could have a null name as validation doesn't check for it.

Reply:
As @aptkingston said if the user's name isn't a feature we need/use could just remove the reference to it being passed in here as a default/required parameter of users to remove any confusion.

Reply:
Yep will just remove it - can always be a custom column if they need it 👍